### DISCUSSION TITLE
Two different CC2531 Sticks with the same configuration to same MQTT broker?

### DISCUSSION THREAD

**Discussion by JsBergbau** [Koenkk/zigbee2mqtt#8624] [General]
I'm using Zigbee2MQTT with several devices.
For testing purposes I've copied the whole /opt/zigbee2mqtt directory to another Raspberry PI and started it while the other instance on first raspberry PI was also running.
I could see the messages of the Zigbee devices on both zigbee2mqtt logs. This second CC231 receiver was only online for a few minutes. 

Since I have some reception problems and a IKEA Zigbee repeater seems to work not so well with Xiaomi Aqara devices, the easiest solution would be to copy the /opt/zigbee2mqtt again on a second raspberry PI and start it. Most devices would be now received by both instances. Would this setup work on the long run? Or should I setup a second Zigbee network with its own encryption key? If so: Can I keep at least the MQTT base topic (zigbee2mqtt), so both publish to the same topic or could this lead to trouble. 

Thanks for sharing your expert knowledge.

**Comment by francisp2** [comment]
This will not work. A device can only be connected to one coordinator at a time. Better flash the second stick with the router firmware.

**Comment by JsBergbau** [comment]
Sadly these aqara devices don't work well with a router. I've already setup one router, but still devices sometimes have connection issues.
